### DISCUSSION TITLE
When to expect official support for 0.69?

### DISCUSSION THREAD

**Discussion by pilotpirxie** [getsentry/sentry-react-native#2448] [General]
It's a few months since RN 0.69 was released and besides 5.0.0-alpha.1 and 2 there is no official and stable sentry plugin :/ Is it already known when to expect official support for 0.69?

**Comment by krystofwoldrich** [comment]
For anyone reading this in the future the support was added in https://github.com/getsentry/sentry-react-native/releases/tag/5.0.0 together with the React Native New Architecture.
